src/inc/ITerminalOwner.hpp

```diff
@@ -1,33 +0,0 @@
-/*++

```

Comment:
So, why don't we need `ITerminalOwner` anymore? Is the short version of why we don't need this is that we're passing the `shutdownEvent` around, so we don't really need to care about their own implemenation of CloseInput/Output?

Reply:
Yes, that's why. We're not grepping around to go find the terminal owner. Anyone stood up to go along with this terminal gets the event and knows to shut themselves down. We don't have to go hunt them down later.